### PR TITLE
Data collection tmu

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
This adds a bunch of stuff. The commit history should be very clean. We now analyzed atop, coreutils and collectd. they introduced new tables. figures and listings. They are now appropriately linked after the table of contents.

File: thesis-de/content-de.tex [hunk 1]
```diff
@@ -477,18 +493,201 @@ \subsubsection{Begriffsklärung}
...
+\subsubsection{Coreutils}
+Coreutils ist eine Sammlung von Standardwerkzeugen auf jedem Linux
+Betriebssystem. Sie bieten einfache Möglichkeiten zur Editierung von Dateien.
+Coreutils ist mit der GPL3 Lizenz lizensiert~\cite{coreutils} und zählt somit
+als Open Source Software. Verwaltet wird die Programmsammlung aktuell von der
+Free Software Foundation. Der Linux Kernel stellt Informationen über
+schreibgesschützte Dateien bereit. In Kombination mit der Programmsammlung
+procps kann auf die Informationen des Kernels zugegriffen werden um Metriken zu
+ermitteln~\cite{procps}. Über das Programm free oder die Datei
+\texttt{/proc/meminfo} erhält man den die Menge des vorhandenen sowie zur Zeit
```
Comment (Marcellii) [inline thesis-de/content-de.tex]: den die Menge

File: thesis-de/content-de.tex [hunk 2]
```diff
@@ -477,18 +493,201 @@ \subsubsection{Begriffsklärung}
...
+benutzten Arbeitsspeichers. Die Anzahl der Prozessorkerne liefert das Programm
+nproc oder die Datei \texttt{/proc/cpuinfo}. \texttt{/proc/diskstats} liefert
+Informationen über die Anzahl der Schreib- und Lesevorgänge auf allen
+Datenträgern. Außerdem listet die Datei die geschriebe als auch gelesene
```
Comment (Marcellii) [inline thesis-de/content-de.tex]: geschriebe -> geschriebene

File: thesis-de/content-de.tex (hunk 1 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

File: thesis-de/content-de.tex (hunk 2 above)
Comment (bastelfreak) [reply thesis-de/content-de.tex]: fixed

**Pull request merged**